triggers/triggers.go

```diff
@@ -5,13 +5,14 @@ package triggers
 import (
 	"bytes"
 	"fmt"
-	htmptemplate "html/template"
+	texttemplate "text/template"
```

Comment:
Switching to text/template fixed all escaping issue and make it opt in via the `html`, `js` and `urlquery` escape function

Reply:
Is it possible to document how this would work then? 😅